app/views/layouts/application.html.erb

```diff
@@ -12,14 +12,16 @@
 
 <script>
   $(function() {
+    var ideas = $('.ideas');
+    var search_bar = $('input[name=search]');
     $('form#createIdea').submit(Ideas.create);
```

Comment:
Same as before, not sure if these selectors need to be saved since they're not repeated.